publiccode.yaml

```diff
@@ -44,6 +44,8 @@ dependsOn:
     - versionMin: 1.0.0
     
 nl:
+  commonground:
+  - layerType: "interface"
```

Comment:
This line can be replaced with `layerType: interface` the reason for that is:
- The `-` suggests that layerType is part of an array which it is not suppose to be
- There is no need for the value to be put into quotations